### PR TITLE
Dashboard v0.01

### PR DESCRIPTION
**Pull request by ChiragSaxena5437**
A very basic dashboard design with minimalistic features.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,13 @@
-import Sidebar from "./components/Sidebar";
```
Comment (rakshit087) [inline src/App.js:1]: Don't rmove the sidebar please xD

**Review by rakshit087 — CHANGES_REQUESTED** [review]
Bro, please pull the latest changes  before PR xD. 
Or ek baar Pulkit ko dekhne dete

File: package.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
 {
-  "name": "karyakram",
+  "name": "karyakaram",
```
Comment (pulkitvyas08) [inline package.json:2]: Wrong name, the name is `karyakram` don't change it

File: package.json [hunk 3]
```diff
@@ -1,20 +1,20 @@
...
-    "@mui/icons-material": "^5.3.0",
+    "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
     "@mui/material": "^5.3.0",
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^12.1.2",
     "@testing-library/user-event": "^13.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-router-dom": "^6.2.1",
```
Comment (pulkitvyas08) [inline package.json:15]: Pull the changes first. @rakshit087 has added router stuff

File: src/App.css [hunk 4]
```diff
@@ -1,6 +1,183 @@
-.App {
```
Comment (pulkitvyas08) [inline src/App.css:1]: Why delete this?

File: src/App.js (hunk 1 above)
Comment (pulkitvyas08) [reply src/App.js:1]: +1

File: src/App.test.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
```
Comment (pulkitvyas08) [inline src/App.test.js:1]: We don't need this atm

**Review by pulkitvyas08 — COMMENTED** [review]
Why is there a submodule?

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]
@ChiragSaxena5437 Quite a few issues here. As @rakshit087 has said, please pull all the changes first then add your stuff or resolve the conflicts

**Comment by ChiragSaxena5437** [comment]
Ok, all the changes made in a new PR.

**Pull request closed without merging**